code-challenges.test.js

```diff
@@ -13,47 +16,151 @@
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
+//input= an object as an argument
+//output= wether the number is evenly divisible by 3
+// i think a conditional statement with the funtion to return wether the number is evenly divisible by three or not is the appropriate approach
 
 // a) Create a test with expect statements for each of the variables provided.
+//input= number
+//output= is divisible by three || is not divisible by three
 
-const object1 = { number: 15 }
+const object1 = { number: 15 };
 // Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
+const object2 = { number: 0 };
 // Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
+const object3 = { number: -7 };
 // Expected output: "-7 is not divisible by three"
+describe("divByThree", () => {
+  it("returns wether the number is divisible by three or not based on input", () => {
+    expect(divByThree(object1)).toEqual("15 is divisible by three");
+    expect(divByThree(object2)).toEqual("0 is divisible by three");
+    expect(divByThree(object3)).toEqual("-7 is not divisible by three");
+  });
+});
 
 // b) Create the function that makes the test pass.
+// i want to make a function inside of a conditional statement i can use for any number. i think it is a called a dynamic code. i will name it divByThree and make the if statement to return if is divisible by three or else return the statement that it is not divisible by three.
+const divByThree = (number) => {
+  if (number % 3 === 0) {
+    return `${number} is divisible by three`;
+  } else if (number % 3 !== 0) {
+    return `${number} is not divisible by three`;
+  }
+};
+// something wrong with jest, or my code.. but test is not running properly. This is what the terminal is returning..
+// FAIL  ./code-challenges.test.js
+// ● Test suite failed to run
+
+// Your test suite must contain at least one test.
+
+//   at onResult (node_modules/@jest/core/build/TestScheduler.js:133:18)
+//   at node_modules/@jest/core/build/TestScheduler.js:254:19
+//   at node_modules/emittery/index.js:363:13
+//       at Array.map (<anonymous>)
+//   at Emittery.emit (node_modules/emittery/index.js:361:23)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       0 total
+// Snapshots:   0 total
+// Time:        0.384 s
+// Ran all test suites.
+// error Command failed with exit code 1.
+// info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
+// it looks like it is stating that i dont have a test? not sure what that means or how to fix it. im thinking that jest must be testing the wrong file. will cd into specific file "code-challenges.test.js and try again"
+// recieved the same message. i hope i can get clearaty for why this isn't working this upcoming week. i must admit, i am not feeling very good about this assesment, because of this result.
```

Comment:
Please make sure to reach out on slack if you run into a blocker like this.  I do see some code on lines 3 and 4 that could be contributing to this problem.